Expensify/App
# [No QA] Add accessibility check to each PR in the deployChecklist

CC: @roryabraham 

### Details
We are adding accessibility to newDot! Because of that we need to update our deploy checklists to accommodate for both QA and accessibility testing.

Accessibility failures will not prevent deploys as of now because nobody is writing accessible code. In the future we will be including accessibility checkboxes as a deploy blocker.

Old Format:
- [x] https://github.com/Expensify/App/pull/5258
- [x] https://github.com/Expensify/App/pull/5262

New Format:
- https://github.com/Expensify/App/pull/5258

- https://github.com/Expensify/App/pull/5262

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/175521

### Tests
SO MANY automated tests, and that's about the best we can do unfortunately. This can't be tested locally.

### QA Steps
1. Make sure the checklist is created in the new format defined above
2. Make sure that when we add deploy blocker labels, it is added to the deploy checklist just like it currently does
3. Make sure that when we CP to staging, they are added to the checklist


### Tested On
n/a

### Screenshots
n/a
